src/UNIV2LPOracle.sol

```diff
@@ -188,10 +188,10 @@ contract UNIV2LPOracle {
         wat  = _wat;
         uint256 dec0 = uint256(ERC20Like(UniswapV2PairLike(_src).token0()).decimals());
         require(dec0 <= 18, "UNIV2LPOracle/token0-dec-gt-18");
-        normalizer0 = 10 ** (18 - dec0);  // Calculate normalization factor of token0
+        
         uint256 dec1 = uint256(ERC20Like(UniswapV2PairLike(_src).token1()).decimals());
         require(dec1 <= 18, "UNIV2LPOracle/token1-dec-gt-18");
-        normalizer1 = 10 ** (18 - dec1);  // Calculate normalization factor of token1
+        normalizer = mul(10 ** (18 - dec1), 10 ** (18 - dec0));  // Calculate normalization factor of token1
```

Comment:
We know this can't overflow so might as well just do `10 ** (36 - dec0 - dec1)`.